pages/organizations/_organizationId/contacts/_contactId/index.vue

```diff
@@ -1,6 +1,15 @@
 <template>
   <div class="container-fluid">
-    <BForm @submit.prevent="save">
+    <BForm ref="contactForm" @submit.prevent="save">
+      <PreventNavigation
+        :prevent="isDirty"
+        :actions="actions"
+        @save="handleSave"
+        @next="handleNext"
```

Comment:
you can simply call $event.next()
```suggestion
        @next="$event.next()"
```

Reply:
DONE! changes done as suggested.